Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterCoordinates.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using Newtonsoft.Json;
+
+namespace Microsoft.Toolkit.Uwp.Services.Twitter
+{
+    public class TwitterCoordinates
+    {
+        private const int LatitudeIndex = 0;
+        private const int LongitudeIndex = 1;
+        private const string PointType = "Point";
+
+        [JsonProperty("coordinates")]
+        public string[] Coordinates { get; set; }
```

Comment:
Can you use a JsonConverter here to do the parsing to make this a named Tuple of double here instead?